### ISSUE TITLE
DocPad Yeoman Generator Direction

### ISSUE THREAD

**Issue by RobLoach** [RobLoach/generator-docpad#22]
@mikeumus pointed out:

> Also Ben and I were chatting about using/expanding @RobLoach's DocPad Yeoman generator so that we don't have skeletons and plugins breaking all the time. Less work for us. This sounds like a priority for me: https://github.com/RobLoach/generator-docpad

Would love to see it expand. How/where do you see it growing?

**Comment by mikeumus** [comment]
In @balupton's words, "we need to pimp it out", so part of that is getting more people aware of it and using it. Does it need any polish? Is it missing anything mission critical? I'm new to Yeoman. I'll start messing with it and your generator today. 

If we apply some polish/docs and consider making it a github/docpad/ project so that more people find it and establish it as the "official" yeoman generator for DocPad (as it seems there were two others in the past) then this is a good start to getting both DocPad users using it and potentially bringing in new users to DocPad through it.

**Comment by balupton** [comment]
@RobLoach as you're a owner for the docpad org, can you transfer this repo over to there?

**Comment by mikeumus** [comment]
So pimping out the Yeoman generator is something that could potentially make everyone's life easier but do we need to reactor into pieces first? 

Like for example making the plugin system a module/product sounds timely/priority.

**Comment by balupton** [comment]
@mikeumus agreed on the priorities, I think though that not everyone can do all tasks. I can split out the plugin system, but @RobLoach would be in a good place for this. Something to discuss in tomorrow's catchup.
